### ISSUE TITLE
Update for Okta?

### ISSUE THREAD

**Issue by k5jj** [stormpath/stormpath-flask#99]
Will this project be forked/updated to work with Okta?    I would really like to see an "okta-flask" project.

**Comment by rdegges** [comment]
Yes. This will be created, although I don't know the exact date right now. This is my personal priority at the company =)

**Comment by dalymople** [comment]
Hi @rdegges, this is great news =)
flask-stormpath has made building my app a breeze!

**Comment by joshford05** [comment]
Hi - is there any update on estimated availability for this or is it still definitely in the works? We're getting close to the Stormpath deprecation date so just trying to figure out if we develop our own wrappers or wait for this release.  Thanks

**Comment by secdec24** [comment]
I contacted Stormpath support earlier this month regarding this same issue and this was their response:

> Thanks for reaching out, and for providing such detailed information about your app. Let me start by providing some context around our current engineering efforts.
> 
> Our plan is to release a new, final version of ASP.NET, ASP.NET Core, Express, SpringBoot, Spring, and Servlet that will be internally patched to work with Okta and your migrated Stormpath data, with as few breaking changes as possible. These final versions are meant to get you onto the Okta platform with as few headaches as possible. However, these libraries will not receive new feature development, as those will go into a new Okta-specific library in the future. This does mean that customers using these integrations will need to refractor around a new library in the future, if those customers need to use a new feature that will go into the new libraries.
> 
> In the case of Flask, there is not a final version of the Flask library being built. This means that you'll be able to export your data from Stormpath and import it into Okta, but will need to build your own integration using Okta's REST API (or use one of the ones we're building -- see the list above) by August 17 when Stormpath shuts down. If your team chooses to migrate to Okta, I strongly recommend converting your code to use one of the integrations above -- ASP.NET, ASP.NET Core, Express, SpringBoot, Spring, or Servlet -- unless you have the resources to build out your own integration. If there are other platforms you're interested in that natively support Flask for authentication/authorization, I would start looking into those as well.
> 
> I wish I had better news for you on the Python front here at Okta. We do plan to eventually build a Python SDK, and a Flask and Django integration, but that work will not be started until after August of this year, and I do not have a firm release date at all. I understand that this news is not coming at a good time for you and your team, and want to apologize for that. If there is anything else I can do to help, please let me know.
